updated irish language webpage.txt

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+<h1 style="text-align:left">Seiceálaí litrithe é seo</h1>
+<h2>
+<center>cuir tús le do théacs anseo</center>
+</h2>
+<br>
+<center>
+<br>
+<form  method="POST">
+					  <textarea  id="content" name = "takeinput" style=" min-width:120px; max-width:100%;min-height:120px;height:100%;width:100%;" ></textarea>

```

Comment:
I might be misunderstanding because I don't know html very well, but this looks like it doesn't include the same text box/button functionality that we have in our current template. Can someone clarify this for me?

Reply:
Can you specify any button which needs to be added?